Get total unread from users.labels.get method instead

iwconfig opened the pull request:
I have a lot of unread messages (shame on me :see_no_evil: :hear_no_evil: :speak_no_evil: :smile:).

The search result in `list` is only the first page. So if you have a lot of unread messages, iterating the result with the `nextPageToken` gives a proper count of the unread messages.

iwconfig committed:
Take nextPageToken into account

The search result in 'list' is only the first page.
So if you have a lot of unread messages, iterating
the result with the 'nextPageToken' gives a proper
count of the unread messages.

vyachkonovalov commented:
Hello, thank you for pull request.
Did you try `maxResults` query option? Cause iterating over pages feels like not so good solution.
btw, how many unread messages you have? :slightly_smiling_face:

iwconfig commented:
Oh, heh, you're absolutely right. Clearly i was over my head with this. I read up on the api documentation a little further and `.users().labels().get(userId='me', id='INBOX').execute()` gives us a dict which contains the `threadsUnread` key which is the total in the 'INBOX' label. This should work. No iteration occurring. 🙂 I'll update this PR soon.

Hrmm.. Heh, right now it's at 511 unread messages. Thats nothing compared to some other email accounts i own. 😄

iwconfig committed:
Read the total unread from labels().get instead

'threadsUnread' key in dict from 'labels().get'
gives the total number of the specific label.
This gives all unread emails and do not iterate
through search results.

Also added a note about labels.
